jest.config.js

```diff
@@ -11,11 +37,11 @@ export default {
     "/node_modules/",
     "/development/",
   ],
+  transformIgnorePatterns: [`/node_modules/(?!${esModules})`],
```

Comment:
```suggestion
  transformIgnorePatterns: [`/node_modules/(?!${esModules})/`],
```

Maybe we should ensure that this is not part of a longer folder name?

Reply:
Works for me!